### PR TITLE
Completed card component

### PR DESCRIPTION
**Pull request by therajatg**
Please review the card component

### CODE REVIEW

File: styles/component-styles/card.css [hunk 1]
```diff
@@ -1,15 +1,65 @@
...
+    justify-content: space-around;
+    word-break: normal;
+}
+
+.horizontal-card>img{
+    height: 100%;
+    width: auto;
+}
+
+
```
Comment (ZuberDunge) [inline styles/component-styles/card.css]: you can remove extra spacing

**Review by ZuberDunge — APPROVED** [review]
Looks good to me.

File: styles/component-styles/card.css (hunk 1 above)
Comment (therajatg) [reply styles/component-styles/card.css]: Done

Comment (therajatg) [reply styles/component-styles/card.css]: done

**Comment by iamsaurav28** [comment]
you have taken display flex many times try to  take it common . or else its good to go

**Review by iamsaurav28 — APPROVED** [review]

**Review by ZuberDunge — APPROVED** [review]

**Pull request merged**